lib/dotcom_web/components/system_status/commuter_rail_status.ex

```diff
@@ -0,0 +1,265 @@
+
+        [{effect, "1 #{effect_string}"}]
+
+      {1, _} ->
+        effect = alert_counts |> Map.keys() |> List.first()
+        effect_string = effect |> Atom.to_string() |> Recase.to_title() |> Inflex.pluralize()
+        count = alert_counts[effect]
+
+        [{effect, "#{count} #{effect_string}"}]
+
+      _ ->
+        [{:alert, "#{total_count} Service Alerts"}]
+    end
+  end
+
+  # Rejects cancellations and delays from the alert counts
+  # so that we can separate cancellations and delays from other alerts.
+  defp reject_cancellations_and_delays(alert_counts) when alert_counts == %{}, do: %{}
+
+  defp reject_cancellations_and_delays(alert_counts) do
+    alert_counts
+    |> Enum.reject(fn {effect, _} -> Enum.member?(~w(cancellation delay)a, effect) end)
+    |> Map.new()
+  end
+
+  # A row that indicates that the service is not running today.
+  # This trumps any other status.
+  defp row(%{row: %{service_today?: false}} = assigns) do
+    ~H"""
+    <a
+      class={[
+        "flex items-center py-2",
+        "hover:bg-brand-primary-lightest cursor-pointer group/row",
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12 text-md md:text-lg"
+      ]}
+      href={@row.url}
+      data-test="status-row"
+    >
+      <div class="grid items-center grid-cols-[min-content_min-content_auto] items-center grow">
+        <div class="flex items-center pl-1 pr-2 min-w-64 md:min-w-72">
+          <span class="text-black">{@row.name}</span>
+        </div>
+
+        <div class="pr-2 flex items-center">
+          <.status_icon status={:no_scheduled_service} />
+        </div>
+
+        <div class="grow flex items-center text-black">
+          No Scheduled Service
+        </div>
+      </div>
+      <div class="self-stretch flex items-center">
+        <.icon name="chevron-right" class="h-3 w-2 fill-gray-dark ml-3 mr-2 shrink-0" />
+      </div>
+    </a>
+    """
+  end
+
+  # The 'normal' case where there are no alerts.
+  # We show a row indicating "Normal Service".
+  defp row(%{row: %{alert_counts: alert_counts}} = assigns) when alert_counts == %{} do
+    ~H"""
+    <a
+      class={[
+        "flex items-center py-2",
+        "hover:bg-brand-primary-lightest cursor-pointer group/row",
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12 text-md md:text-lg"
+      ]}
+      href={@row.url}
+      data-test="status-row"
+    >
+      <div class="grid items-center grid-cols-[min-content_min-content_auto] items-center grow">
+        <div class="flex items-center pl-1 pr-2 min-w-64 md:min-w-72">
+          <span>{@row.name}</span>
+        </div>
+
+        <div class="pr-2 flex items-center">
+          <.status_icon status={:normal} />
+        </div>
+
+        <div class="grow flex items-center">
+          <.status_label status={:normal} />
```

Comment:
This doesn't need any change (compared to "No service today", which does) because `<.status_label />` already renders correctly.